tests/primary.test.mjs

```diff
@@ -412,4 +412,29 @@ describe('primary names', function () {
       });
     });
   });
+
+  describe('getPaginatedPrimaryNameRequests', function () {
+    it('should return all primary name requests', async function () {
+      const getPaginatedPrimaryNameRequestsResult = await handle({
+        Tags: [
+          { name: 'Action', value: 'Primary-Name-Requests' },
+          { name: 'Limit', value: 10 },
+          { name: 'Sort-By', value: 'startTimestamp' },
+          { name: 'Sort-Order', value: 'asc' },
+        ],
+      });
+      assertNoResultError(getPaginatedPrimaryNameRequestsResult);
+      const primaryNameRequests = JSON.parse(
+        getPaginatedPrimaryNameRequestsResult.Messages[0].Data,
+      );
+      assert.deepStrictEqual(primaryNameRequests, {
+        items: [],
+        totalItems: 0,
```

Comment:
I can haz 1 item and a cursor plz?

Reply:
Unit tests look alright. Might be nice to see cursor and a sortBy preference exercised here though.